README.md

```diff
@@ -179,6 +179,7 @@ npx -y @smithery/cli install @shariqriazz/vertex-ai-mcp-server --client claude
     }
```

Comment:
`VERTEX_AI_MODEL_ID` variable (line 163) should be changed to the proper value used in config: `VERTEX_MODEL_ID`. Current readme is misleading.